static/src/javascripts/projects/common/modules/experiments/tests/contributions-story.js

```diff
@@ -0,0 +1,176 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-story.html',
+    'text!common/views/contributions-epic.html',
+    'common/utils/robust',
+    'inlineSvg!svgs/icon/arrow-right',
+    'common/utils/config',
+    'common/modules/commercial/commercial-features',
+    'common/utils/cookies',
+    'common/utils/element-inview'
+], function (bean,
+             qwery,
+             $,
+             template,
+             svg,
+             fastdom,
+             mediator,
+             contributionsStory,
+             contributionsEpic,
+             robust,
+             arrowRight,
+             config,
+             commercialFeatures,
+             cookies,
+             ElementInview
+) {
+
+    return function () {
+
+        this.id = 'ContributionsStory20160922';
+        this.start = '2016-09-22';
+        this.expiry = '2016-10-03';
+        this.author = 'Jonathan Rankin';
+        this.description = 'Test whether telling the story of the guardian through staggered messages over time results in more contributions than always showing the epic message.';
+        this.showForSensitive = false;
+        this.audience = 0.08;
+        this.audienceOffset = 0.48;
+        this.successMeasure = 'Impressions to number of contributions';
+        this.audienceCriteria = 'Non members viewing the UK edition';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'Telling the story of the guardian through staggered messages over time result in an uplift of 30% in impressions to conversions';
+        this.canRun = function () {
+            var worksWellWithPageTemplate = (config.page.contentType === 'Article'); // may render badly on other types
```

Comment:
That seems rather an arbitrary decision. Why would you prevent them showing up on the Chirldren's book site, but be fine with them showing up on say https://www.theguardian.com/books/2016/sep/24/childrens-books-roundup-best-picture-books-novels. 

If this is a decision we're making about particular pieces of content, surely that should be reflected in the information returned by CAPI, rather than rules in frontend.